pkg/testing/fixture_install.go

```diff
@@ -174,3 +200,34 @@ func (f *Fixture) Uninstall(ctx context.Context, uninstallOpts *UninstallOpts, o
 
 	return out, nil
 }
+
+func (f *Fixture) collectDiagnostics() {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	dir, err := findProjectRoot(f.caller)
+	if err != nil {
+		f.t.Logf("failed to collect diagnostics; failed to find project root: %s", err)
+		return
+	}
+	diagPath := filepath.Join(dir, "build", "diagnostics")
+	_ = os.MkdirAll(diagPath, 0755)
```

Comment:
[Blocker]
Why are you ignoring the error? If it fails there will be a cascading effect, better to handle it and stop in case the directory cannot be created.

Reply:
Fixed.